### ISSUE TITLE
hi， i only have a titan for 12G,can i run the train.py? 

### ISSUE THREAD

**Issue by dzz416** [JihyongOh/XVFI#6]

**Comment by hjSim** [comment]
Yes, you can. We also have tried to train the network with NVIDIA Titan Xp and the memory usage was about 7GB for patch size 384x384 and 11GB for patch size 512x512. All other experiment settings are the same as the description in our paper.

**Comment by dzz416** [comment]
> Yes, you can. We also have tried to train the network with NVIDIA Titan Xp and the memory usage was about 7GB for patch size 384x384 and 11GB for patch size 512x512. All other experiment settings are the same as the description in our paper.

I have a another problem, when i download the encode_train.tar  ,it always failed in the end, can i download it in another way ?

**Comment by hjSim** [comment]
Since the uncompressed original training dataset (as .png files) is about 240GB, we could not upload the uncompressed one online.
What is the error when you run 'mp4_decoding.py' we provided?

**Comment by dzz416** [comment]
The training set in the link is 14g, and every time I download it to 13.9g, it is banned.

**Comment by JihyongOh** [comment]
@dzz416 ,
We have successfully re-downloaded it through the same link just now. 
Can you check your network condition, please?

**Comment by dzz416** [comment]
I have downloaded the train data successfully,thank you ! I 'm just getting start , Can you tell me the exact format of the training input data and output data of this network model and the function of decode.py?tanks again

**Comment by dzz416** [comment]
And how long does it take to train?

**Comment by hjSim** [comment]
@dzz416 
We use '.png' image file for both input and output data. 
'.png' files are extracted from the encoded files we provided, via dropbox link. 
'mp4_decoding.py' file can be found in the same dropbox link as well.
Training takes about a half-day with an NVIDIA TITAN RTX as described in our paper. It depends on the experiment system.
Most of information can be found in our paper or github readme page
Thank you!
